Symfony/CS/Tests/Console/ConfigurationResolverTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\CS\Tests\Console;
 
+use Guzzle\Common\Exception\UnexpectedValueException;
```

Comment:
sorry, but I can't see where (and why) `ConfigurationResolver` throws exception from `Guzzle` namespace